soc/arm/nuvoton_npcx/common/soc_pins.h

```diff
@@ -36,6 +36,19 @@ struct npcx_lvol {
 	uint16_t bit:3; /** Related register bit for low-voltage conf. */
 };
 
+/**
+ * @brief NPCX Power Switch Logic (PSL) input configuration structure
+ *
+ * Used to configure PSL input pad which detect the wake-up events and switch
+ * core power supply (VCC1) on from standby power state (ultra-low power mode).
+ */
+struct npcx_psl_in {
+	uint32_t flag; /** flag to indicate the detection mode and type. */
+	uint32_t offset; /** offset in PSL_CTS for status and detection mode. */
+	struct npcx_alt pinctrl; /** DEVALT register/bit for PSL pin-muxing. */
```

Comment:
What does DEVALT mean? Can you add a comment about it above?

Reply:
Done.